platforms/anchor-platform/admin-guide/sep24/faq.mdx

```diff
@@ -3,16 +3,16 @@ title: "FAQ"
 sidebar_position: 50
 ---
 
-### How To Use Interactive JWTs?
+### How To Use JWTs?
 
-As part of the flow, once a user makes an interactive withdrawal/deposit request, it will be processed by the Anchor Platform and forwarded to your service. The Anchor Platform will make a `GET` call to `<configured interactive url>?token=<jwt token>`.
+As part of the flow, once a user makes a request, i.e. an interactive withdrawal/deposit request, it will be processed by the Anchor Platform and forwarded to your service. The Anchor Platform will make a `GET` call to `<configured url>?token=<jwt token>`.
```

Comment:
```suggestion
As part of the interactive flow, once a user makes a request, i.e. an interactive withdrawal/deposit request, it will be processed by the Anchor Platform and forwarded to your service. The Anchor Platform will make a `GET` call to `<configured interactive url>?token=<jwt token>`.
```